### PR TITLE
translate: introduction & download page

### PR DESCRIPTION
**Pull request by Showave**
Translate [Introduction]、[Download] page, issue: #109 
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by Showave** [comment]
PTAL, @imbajin , thanks

File: content/en/docs/download/download.md [hunk 1]
```diff
@@ -10,10 +10,10 @@ The latest HugeGraph: **0.12.0**, released on _2021-12-31_.
...
-HugeGraph-Hubble | 基于Web的可视化图形界面  | [1.6.0](https://github.com/hugegraph/hugegraph-hubble/releases/download/v1.6.0/hugegraph-hubble-1.6.0.tar.gz)
-HugeGraph-Loader | 数据导入工具            | [0.12.0](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.12.0/hugegraph-loader-0.12.0.tar.gz)
-HugeGraph-Tools  | 命令行工具集            | [1.6.0](https://github.com/hugegraph/hugegraph-tools/releases/download/v1.6.0/hugegraph-tools-1.6.0.tar.gz)
+HugeGraph-Server | The main program of HugeGraph      | [0.12.0](https://github.com/hugegraph/hugegraph/releases/download/v0.12.0/hugegraph-0.12.0.tar.gz)
+HugeGraph-Hubble | Web-based Visual Graphical Interface  | [1.6.0](https://github.com/hugegraph/hugegraph-hubble/releases/download/v1.6.0/hugegraph-hubble-1.6.0.tar.gz)
+HugeGraph-Loader | Data import tool            | [0.12.0](https://github.com/hugegraph/hugegraph-loader/releases/download/v0.12.0/hugegraph-loader-0.12.0.tar.gz)
+HugeGraph-Tools  | command line toolset            | [1.6.0](https://github.com/hugegraph/hugegraph-tools/releases/download/v1.6.0/hugegraph-tools-1.6.0.tar.gz)
```
Comment (imbajin) [inline content/en/docs/download/download.md:16]: `Command`

File: content/en/docs/download/download.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ server
 [0.5.6](https://hugegraph.github.io/hugegraph-downloads/hugegraph-release-0.5.6-SNAPSHOT.tar.gz) | [1.5.0](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-client/1.5.0)  | [0.5.6](https://hugegraph.github.io/hugegraph-downloads/hugegraph-loader/hugegraph-loader-0.5.6-bin.tar.gz)     | &#124; [0.5.0](https://hugegraph.github.io/hugegraph-downloads/hugegraph-studio/hugestudio-release-0.5.0-SNAPSHOT.tar.gz) | 1.4.0  |
 [0.4.5](https://hugegraph.github.io/hugegraph-downloads/hugegraph-release-0.4.5-SNAPSHOT.tar.gz) | [1.4.7](https://mvnrepository.com/artifact/com.baidu.hugegraph/hugegraph-client/1.4.7)  | [0.2.2](https://hugegraph.github.io/hugegraph-downloads/hugegraph-loader/hugegraph-loader-0.2.2-bin.tar.gz)     | &#124; [0.4.1](https://hugegraph.github.io/hugegraph-downloads/hugegraph-studio/hugestudio-release-0.4.1-SNAPSHOT.tar.gz) | 1.3.12 |
 
-> 说明：最新的图分析和展示平台为 hubble，支持 0.10 及之后的 server 版本；studio 为 server 0.10.x 以及之前的版本的图分析和展示平台，其功能从 0.10 起不再更新。
```
Comment (imbajin) [inline content/en/docs/download/download.md:32]: remove this both in chinese and english version (in apache version we should delete it) 

and remove all of the `studio` link is better

File: content/en/docs/introduction/README.md [hunk 3]
```diff
@@ -6,60 +6,58 @@ weight: 1
...
+It can be integrated with big data platforms such as Hadoop and Spark for offline analysis (OLAP).
 
-HugeGraph典型应用场景包括深度关系探索、关联分析、路径搜索、特征抽取、数据聚类、社区检测、
-知识图谱等，适用业务领域有如网络安全、电信诈骗、金融风控、广告推荐、社交网络和智能机器人等。
+Typical application scenarios of HugeGraph include deep relationship exploration, association analysis, path search, feature extraction, data clustering, community detection, knowledge graph, etc., and are applicable to business fields such as network security, telecommunication fraud, financial risk control, advertising recommendation, social network and intelligence Robots etc.
 
-本系统的主要应用场景是解决百度安全事业部所面对的反欺诈、威胁情报、黑产打击等业务的图数据存储和建模分析需求，在此基础上逐步扩展及支持了更多的通用图应用。
+Typical application scenarios of HugeGraph include deep relationship exploration, association analysis, path search, feature extraction, data clustering, community detection, knowledge graph, etc., and are applicable to business fields such as network security, telecommunication fraud, financial risk control, advertising recommendation, social network and intelligence Robots etc.
```
Comment (imbajin) [inline content/en/docs/introduction/README.md:16]: rebase/update with the latest `website` branch to solve the conflict

File: content/en/docs/introduction/README.md [hunk 4]
```diff
@@ -6,60 +6,58 @@ weight: 1
...
-- 技术负责人：[李章梅](https://github.com/javeme)
-- 接口人：[张义](https://github.com/zhoney)，[李凝瑞](https://github.com/Linary)
-- 反馈邮箱：[hugegraph@googlegroups.com](mailto:hugegraph@googlegroups.com)
-- 微信公众号：HugeGraph
+- Project sponsor: Liu Jie
+- Project leader: Han Zuli
+- Technical person in charge: [Li Zhangmei](https://github.com/javeme)
+- Interface person: [Zhang Yi](https://github.com/zhoney), [Li Ningrui](https://github.com/Linary)
+- Feedback Email: [hugegraph@googlegroups.com](mailto:hugegraph@googlegroups.com)
+- WeChat public account: HugeGraph
```
Comment (imbajin) [inline content/en/docs/introduction/README.md:63]: rebase with the latest branch, the info have been updated

**Review by imbajin — COMMENTED** [review]
Thanks, please rebase the branch first, then remove all `studio` relative info

File: content/en/docs/download/download.md (hunk 1 above)
Comment (Showave) [reply content/en/docs/download/download.md]: done, thanks

File: content/en/docs/introduction/README.md (hunk 3 above)
Comment (Showave) [reply content/en/docs/introduction/README.md:16]: done

File: content/en/docs/download/download.md (hunk 2 above)
Comment (imbajin) [reply content/en/docs/download/download.md:32]: image

we should keep the older link for users to download studio, keep the links < 0.11

Comment (Showave) [reply content/en/docs/download/download.md:32]: done

**Comment by imbajin** [comment]
Thanks for your contribution~

**Pull request merged**